CHANGELOG.md

```diff
@@ -10,6 +10,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
  - API: New function to decode the image using a callback outputting a part of a
    row per call.
  - API: 16-bit float output support.
+ - API: JxlDecoderRewind and JxlDecoderSkipFrames functions to skip more
```

Comment:
nit: we used backticks (\`) around these code function names.

Reply:
Done